storage/raft_log_queue_test.go

```diff
@@ -88,6 +88,7 @@ func TestGetTruncatableIndexes(t *testing.T) {
 	// Enable the raft log scanner and and force a truncation.
 	store.DisableRaftLogQueue(false)
 	store.ForceRaftLogScanAndProcess()
+	stopper.Quiesce()
```

Comment:
can you put a comment above this line explaining why it's done? This is only the second location in the codebase (outside of stopper.go) that this method is called.